components/HeaderContainer.tsx

```diff
@@ -1,15 +1,33 @@
 import React from "react";
-import styles from "./HeaderContainer.module.css";
-import sharedStyles from "./treeView.module.css";
-import org from "../config.json";
-
+import headerStyles from "./HeaderContainer.module.css";
+// import sharedStyles from "./treeView.module.css";
+import config from "../config.json";
+import ForestIcon from "@mui/icons-material/Forest";
+import { RepoOverviewCondensedProps } from "./SummaryComponents/CondensedSummary/CondensedSummary";
 export default function HeaderContainer() {
```

Comment:
I don't think this function / file is in use; I added the header section to the summary container but didn't remove this file whoops (think it has been removed in another one of my branches though).